zerver/models.py

```diff
@@ -2570,6 +2570,13 @@ def where_unread() -> str:
         # the code for example usage.
         return "flags & 1 = 0"
 
+    @staticmethod
+    def where_read() -> str:
+        # Use this for Django ORM queries to access read messages.
+        # This custom SQL plays nice with our partial indexes.  Grep
+        # the code for example usage.
+        return "flags & 1 <> 0"
```

Comment:
How did you confirm that this works with the partial indexes?  It seems very possible to me that it does not and we need to create an index for this purpose.

Reply:
That said, I don't think we need an index for this operation -- we're fetching with a given `message_id`, which does have a nice index, and it's fine to scan all the matches.  So I think we should just replace the comment with one warning that this operation likely does not have an index, so should only be used for queries where another good index is already used.

Reply:
Oh, I'm not very familiar with partial indices. I thought this would be the same as `where_unread`, and got the comment from it. Will change the comment and spend a bit of time reading about partial indices. Thanks! :slightly_smiling_face:

Reply:
Changed the comment to
```
        # It is likely that this operation doesn't have an index. So,
        # use this only for queries where another good index is present.